### PR TITLE
Feature/update prettier config

### PR DESCRIPTION
**Pull request by SuperITMan**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [X] The commit message follows our guidelines: https://github.com/NationalBankBelgium/code-style/blob/master/CONTRIBUTING.md#-commit-message-guidelines
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Only the configuration for Prettier 1.16.x is available.

Issue Number: #13 

## What is the new behavior?

Configurations for versions `1.17.x`, `1.18.x`, `1.19.x` and `2.0.x` have been added.

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

## Other information

The only relevant change is the fact that I replaced the file `.prettierrc.js` by `.prettierrc` and the content from

```js
module.exports = require("@nationalbankbelgium/code-style/prettier/1.16.x");
```

```txt
"@nationalbankbelgium/code-style/prettier/1.17.x"
```

The same change has been applied in the README files 😊

### CODE REVIEW

File: lib/prettier/2.0.x/.prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module.exports = {
```
Comment (christophercr) [inline lib/prettier/2.0.x/.prettierrc.js:1]: Also the [default value for `endOfLine` changed from `auto` to `lf`](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-endofline-to-lf-7435httpsgithubcomprettierprettierpull7435-by-kachkaevhttpsgithubcomkachkaev). 

Not sure whether we should use the default or set `auto` to keep the old behavior.

In fact we have this issue open related to this: #16 

I think we should test this very well before if we decide to switch to `lf`...

[the remark this answers is not in the document]
Comment (SuperITMan) [reply lib/prettier/2.0.x/.prettierrc.js:1]: I have the feeling that the default value should be ours, based on our codebase. I set `"arrowParens": "always"` :blush:

File: lib/prettier/README.md [hunk 2]
```diff
@@ -9,13 +9,14 @@ This sub-package hosts the [Prettier](https://prettier.io) configuration.
...
 
 ## Usage
 
-Create a `.prettierrc.js` file and copy the following content inside:
+Create a `.prettierrc` file and copy the following content inside:
 
-```js
-module.exports = require("@nationalbankbelgium/code-style/prettier/1.16.x");
+```txt
+"@nationalbankbelgium/code-style/prettier/1.17.x"
```
[the remark this answers is not in the document]
Comment (SuperITMan) [reply lib/prettier/README.md]: I revert this change. Thanks for your remark

File: lib/prettier/2.0.x/.prettierrc.js (hunk 1 above)
Comment (SuperITMan) [reply lib/prettier/2.0.x/.prettierrc.js:1]: Actually, you're right. We should keep `auto` as it is the default behavior for Git @NBB.
I'll set `auto` for configurations > 2.0.x. 

Thanks for your remark :+1:

**Review by nicanac — APPROVED** [review]
Seems good tome. Tried on two projects, it's running smoothly

**Pull request merged**